src/Apps/Sell/Components/SubmissionHeader.tsx

```diff
@@ -73,14 +73,30 @@ export const SubmissionHeader: React.FC = () => {
                   </Media>
 
                   {submissionID && !isLastStep ? (
-                    <Button
-                      variant="tertiary"
-                      onClick={handleSaveAndExit}
-                      display="block"
-                      loading={isSubmitting}
-                    >
-                      Save & Exit
-                    </Button>
+                    <>
+                      <Media at="xs">
+                        <RouterLink
```

Comment:
Using a RouterLink on mobile to not have invisible padding around the text (`variant="tertiary"`).